src/lib/hmacmd5.c

```diff
@@ -27,6 +27,11 @@
 
 RCSID("$Id$")
 
+#ifdef HAVE_OPENSSL_EVP_H
```

Comment:
I guarded around `HAVE_OPENSSL_EVP_H`; would you prefer I guard around something else?